p2panda-net-next/src/actors/subscription.rs

```diff
@@ -39,25 +60,37 @@ impl Actor for Subscription {
         myself: ActorRef<Self::Msg>,
         endpoint: Self::Arguments,
     ) -> Result<Self::State, ActorProcessingErr> {
+        let public_key_suffix = to_public_key(endpoint.node_id()).to_hex()[..6].to_string();
+
         // Spawn the gossip actor.
         let (gossip_actor, _) = Actor::spawn_linked(
-            Some("gossip".to_string()),
+            Some(with_suffix("gossip", &public_key_suffix)),
```

Comment:
follow up: These actor identifiers feel like constants and should maybe be better expressed as such